security/pkg/nodeagent/sds/server.go

```diff
@@ -163,6 +178,48 @@ func NewPlugins(in []string) []plugin.Plugin {
 	return plugins
 }
 
+func (s *Server) initDebugServer(options *Options) {
+
+	mux := http.NewServeMux()
+	mux.HandleFunc(fmt.Sprintf("%s/%s/%s", debugBase, "sds", "workload"), s.sdsWorkloadDebug)
+	mux.HandleFunc(fmt.Sprintf("%s/%s/%s", debugBase, "sds", "gateway"), s.sdsGatewayDebug)
+	s.debugServer = &http.Server{
+		Addr:    fmt.Sprintf(":%d", debugPort),
+		Handler: mux,
+	}
+
+	go func() {
+		err := s.debugServer.ListenAndServe()
+		log.Errorf("%s", err)
+	}()
+}
+
+func (s *Server) sdsWorkloadDebug(w http.ResponseWriter, req *http.Request) {
+	w.Header().Add("Content-Type", "application/json")
+
+	workloadJSON, err := s.workloadSds.DebugInfo()
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		failureMessage := fmt.Sprintf("internal failure: %s", err)
+		w.Write([]byte(failureMessage))
+		return
+	}
+	w.Write([]byte(workloadJSON))
+}
+
+func (s *Server) sdsGatewayDebug(w http.ResponseWriter, req *http.Request) {
+	w.Header().Add("Content-Type", "application/json")
+
+	gatewayJSON, err := s.gatewaySds.DebugInfo()
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
```

Comment:
just want to make sure, by default the response code is 200?

Reply:
Just checked, looks like it defaults to 200